README_fr.md

```diff
@@ -0,0 +1,63 @@
+# CodiMD pour YunoHost
+
+[![Niveau d'intégration](https://dash.yunohost.org/integration/codimd.svg)](https://dash.yunohost.org/appci/app/codimd) ![](https://ci-apps.yunohost.org/ci/badges/codimd.status.svg) ![](https://ci-apps.yunohost.org/ci/badges/codimd.maintain.svg)  
+[![Installer CodiMD avec YunoHost](https://install-app.yunohost.org/install-with-yunohost.png)](https://install-app.yunohost.org/?app=codimd)
+
+*[Read this readme in english.](./README.md)* 
+
+> *Ce package vous permet d'installer CodiMD rapidement et simplement sur un serveur Yunohost.  
+Si vous n'avez pas YunoHost, consultez [le guide](https://yunohost.org/#/install) pour apprendre comment l'installer.*
+
+## Vue d'ensemble
+CodiMD est un service web de traitement de texte collaboratif en temps réel. Il utilise le langage Markdown.
+
+**Version incluse :** 1.6.0
+
+## Captures d'écran
+
+![](https://demo.codimd.org/screenshot.png)
+
+## Démo
+
+* [Démo officielle](https://demo.codimd.org/)
+
+## Configuration
+
+Vous pouvez configurer CodiMD en modifiant le fichier `/var/www/codimd/config.json` et en vous aidant de la [documentation](https://github.com/codimd/server/blob/master/docs/configuration-config-file.md)
+
+## Documentation
+
+ * Documentation officielle : https://hackmd.io/c/codimd-documentation/
```

Comment:
Sorry, but this will only cause confusion. We don't host any documentation on HackMD.io. (This documentation is for the HackMD team version of CodiMD and unreleated to the fork)

As above, please use https://github.com/codimd/server/tree/master/docs/

Or point it to a version specific documentation (e.g. https://github.com/codimd/server/tree/1.6.0/docs/) :)

Reply:
Ok, my bad, I probably picked the wrong documentation or didn't change it when the project was forked.

Reply:
documentation link changed to https://github.com/codimd/server/tree/master/docs/